sysapi/sysapi_util/CommandUtil.c

```diff
@@ -159,16 +158,11 @@ TSS2_RC CommonComplete(_TSS2_SYS_CONTEXT_BLOB *ctx)
         rval = Tss2_MU_UINT32_Unmarshal(ctx->cmdBuffer,
                                         ctx->maxCmdSize,
                                         &ctx->nextData,
-                                        &ctx->rpBufferUsedSize);
+                                        NULL);
         if (rval)
```

Comment:
No need for this check here

Reply:
1. I don't agree, since the TPM response comes from outside. An 12-byte TPM response would make this call fail.
1. More general: We'd end up with a unused variable or unchecked return value which is unnice. So I personally favour all returns a checked, even if it is unused.

@tstruk I'd like to leave the check in, if that's ok with you.

Reply:
I don't see it as unchecked return value because in either case it is up to the caller of CommonComplete() to check it. In my opinion return rval; followed by return rval; doesn't make much sense. And we can always remove the rval if it is not needed.

Reply:
Having said that I like this change and I can remove the redundant check and the unused variable later.

Reply:
Oh, sorry... I completely missed out on the double returning...

I guess I'd personally leave it in, just because people might add something before the return. But go ahead and make your choice. I don't care...

Reply:
I wouldn't worry about the future. If people will change it then we will see if that's ok or not.

Reply:
ok, changed and pushed...

Reply:
If you have pushed to the old PR then it's too late because it has been merged already.
Could you please create a new PR for the new change?

Reply:
sone #644